packages/react/src/components/form/authenticating/messages.ts

```diff
@@ -1,19 +1,48 @@
 import { Factor } from "@slashid/slashid";
 import { TextConfigKey } from "../../text/constants";
+import { Handle } from "../../../domain/types";
 
 type AuthenticatingMessageOptions = {
   isSubmitting: boolean;
   hasRetried: boolean;
 };
 
+/**
+ *  Helper function to extract phone number or email address if either is present in the handle.
+ *
+ * @param {Handle | undefined} handle Handle used for authentication. Can be undefined (e.g. when
+ * authenticating with OIDC).
+ * @returns {Record<string,string> | undefined} tokens map or undefined
+ */
+export function getTokensFromHandle(
```

Comment:
`token` is an overloaded term, maybe this would be more accurate: `getTextTokensFromHandle`?